app/seeders/review_seeder.rb

```diff
@@ -0,0 +1,25 @@
+class ReviewSeeder
+  REVIEWS = [
+    {
+        dinosaur: Dinosaur.first,
+        user: User.first,
+        rating: 5,
+        body: "Giga is the coolest, he's totes my favs"
+    }, {
+        dinosaur: Dinosaur.first,
+        user: User.first,
+        rating: 1,
+        body: "Giga is the worst, he's totes not my favs"
+    }, {
+        dinosaur: Dinosaur.last,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.